store/src/test/java/org/apache/rocketmq/store/dledger/DLedgerCommitlogTest.java

```diff
@@ -65,34 +65,43 @@ public void testTruncateCQ() throws Exception {
             messageStore.shutdown();
         }
 
-        {
-            //Abnormal recover, left some commitlogs
-            DefaultMessageStore messageStore = createDledgerMessageStore(base, group, "n0", peers, null, true, 4);
-            DLedgerCommitLog dLedgerCommitLog = (DLedgerCommitLog) messageStore.getCommitLog();
-            DLedgerServer dLedgerServer = dLedgerCommitLog.getdLedgerServer();
-            DLedgerMmapFileStore dLedgerMmapFileStore = (DLedgerMmapFileStore) dLedgerServer.getdLedgerStore();
-            MmapFileList mmapFileList = dLedgerMmapFileStore.getDataFileList();
-            Thread.sleep(1000);
-            Assert.assertEquals(20, mmapFileList.getMappedFiles().size());
-            Assert.assertEquals(0, messageStore.getMinOffsetInQueue(topic, 0));
-            Assert.assertEquals(1700, messageStore.getMaxOffsetInQueue(topic, 0));
-            Assert.assertEquals(0, messageStore.dispatchBehindBytes());
-            doGetMessages(messageStore, topic, 0, 1700, 0);
-            messageStore.shutdown();
+        try {
```

Comment:
It is not a good way to add exception capture in the test case, it is best to find the cause of the error and fix it.

Reply:
thanks for your suggestions, I will fix it

Reply:
![2021-07-14_190517](https://user-images.githubusercontent.com/38873832/125612285-93e31f56-da01-4ff3-a6f6-7357d29cae53.jpg)
In linux file.delete() can delete file even if the file is in mmap, but in windows it can't. I'm not familiar with this. Do you have any suggestion?